docs/user-guide/deploying-applications.md

```diff
@@ -0,0 +1,95 @@
+# Kubernetes User Guide: Managing Applications: Deploying continuously running applications
+
+You previously read about how to quickly deploy a simple replicated application using [`kubectl run`](quick-start.md) and how to configure and launch single-run containers using pods (configuring-containers.md). Here, you’ll use the configuration-based approach to deploy a continuously running, replicated application.
```

Comment:
In one place you call it declarative and here you call it configuration-based?  Or are those subtly different?